src/Spryker/Zed/EventBehavior/Business/Model/EventResourceQueryContainerManager.php

```diff
@@ -92,7 +86,19 @@ protected function triggerEvents(EventResourceQueryContainerPluginInterface $plu
     protected function triggerEventsAll(EventResourceQueryContainerPluginInterface $plugin): void
     {
         $query = $plugin->queryData();
```

Comment:
Isn't it the same as:

```
       $query = $plugin->queryData();

        if !($query ||  !$query->count()) {
            $this->trigger($plugin, [static::ID_NULL]);

             return;
        }
```

Reply:
No, before there was this `if` https://github.com/spryker/event-behavior/pull/34/files/0b3d4acdf65a0b5bfa25eb474a586c1c9d5439d3#diff-54186e327cb2685a6a326c8715e80aa3L78
that "guaranteed", that `->count()` won't be called on NULL